nipype/info.py

```diff
@@ -141,7 +141,8 @@ def get_nipype_gitversion():
     'configparser',
     'pytest>=%s' % PYTEST_MIN_VERSION,
     'mock',
-    'pydotplus'
+    'pydotplus',
+    'packaging',
```

Comment:
you can do this inside a basic continuumio/miniconda docker instance - think of that as a virtualenv.

```
docker run -it --rm -v /path/to/nipype:/nipype-src continuumio/miniconda bash
> cd /nipype-src
> python setup.py bdist_wheel
```